ChangeLog

```diff
@@ -16,6 +16,10 @@ Pylint's ChangeLog
 
 * Add missing checks for deprecated functions.
 
+* Reduce usage of blacklist/whitelist terminology. Note that the option `extension-pkg-whitelist` is
+  deprecated in favor of `extension-pkg-allow-list`, and the message `blacklisted-name` is now
+  emitted as `disallowed-name`.
+
```

Comment:
```suggestion
* Reduce usage of blacklist/whitelist terminology. The option  `extension-pkg-allow-list` can be used instead of 
`extension-pkg-whitelist` and the message `blacklisted-name` is now emitted as `disallowed-name` (all disable will 
still works).

```

Reply:
Sorry, I missed this suggested change and applied a different edit when removing indications of deprecation. If you prefer your suggestion I'm happy to apply it.